staging/com.scoreSmart/manifest.tt

```diff
@@ -1,26 +1,31 @@
-class @com.scoreSmart {
+class @com.goal_smart {
   import loader from @org.thingpedia.v2();
   import config from @org.thingpedia.config.none();
+  import config from @org.thingpedia.config.none(api_key=$);
+  
+  
+  entity soccer_team #_[description="Soccer Teams in the World"];
+  entity soccer_league #_[description="Soccer Leagues in the World"];
 
   monitorable list query standings(in req league_id: String #_[prompt="what league standings do you want out check out?"],
-                        out Team: String,
-                        out Rank: Number,
-                        out Form: String,
-                        out Points: Number)
-  #_[canonical="get league standings"]
+                        out team: String,
```

Comment:
Given you declared the entity, this should be `out team : Entity(com.goal_smart:soccer_team)` (and similarly the other parameters)